modules/terminate-workers/versions.tf

```diff
@@ -0,0 +1,7 @@
+terraform {
+  required_version = ">= 0.13"
+  required_providers {
+    aws   = ">= 2.46"
+    local = ">= 1.4"
```

Comment:
question: Is this provider still in use?

Reply:
Nope, removed in a290efa4934c4a43c3f14143be6f21d5ba1ef689